src/components/views/groups/GroupRoomList.js

```diff
@@ -120,25 +129,33 @@ export default React.createClass({
             return null;
         }
 
+        let inviteButton;
+        if (GroupStore.isUserPrivileged(this.props.groupId)) {
+            inviteButton = (<AccessibleButton className="mx_RightPanel_invite" onClick={this.onAddRoomToGroupButtonClick}>
+                    <div className="mx_RightPanel_icon" >
+                        <TintableSvg src="img/icons-room-add.svg" width="18" height="14" />
+                    </div>
+                    <div className="mx_RightPanel_message">{ _t('Add rooms to this community') }</div>
+                </AccessibleButton>);
+        }
         const inputBox = (
-            <form autoComplete="off">
```

Comment:
don't we want to keep this?

Reply:
We do! Added it back on the input element.